### PR TITLE
Add Kotlin code examples for writing tests topics

### PR DESCRIPTION
**Pull request by danil-pavlov**
Adds Kotlin tabbed code snippets to three more topics in the Writing Tests section of the User Guide,
continuing the effort started in https://github.com/junit-team/junit-framework/pull/5316, https://github.com/junit-team/junit-framework/pull/5432, and https://github.com/junit-team/junit-framework/pull/5590

Pages updated (Writing Tests):

* Assertions
* Disabling Tests
* Parallel Execution
---

I hereby agree to the terms of the [JUnit Contributor License Agreement](https://github.com/junit-team/junit-framework/blob/002a0052926ddee57cf90580fa49bc37e5a72427/CONTRIBUTING.md#junit-contributor-license-agreement).

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Method [preconditions](https://docs.junit.org/snapshot/api/org.junit.platform.commons/org/junit/platform/commons/util/Preconditions.html) are checked and documented in the method's Javadoc
- [x] [Coding conventions](https://github.com/junit-team/junit-framework/blob/HEAD/CONTRIBUTING.md#coding-conventions) (e.g. for logging) have been followed
- [x] Change is covered by [automated tests](https://github.com/junit-team/junit-framework/blob/HEAD/CONTRIBUTING.md#tests) including corner cases, errors, and exception handling
- [x] Public API has [Javadoc](https://github.com/junit-team/junit-framework/blob/HEAD/CONTRIBUTING.md#javadoc) and [`@API` annotations](https://apiguardian-team.github.io/apiguardian/docs/current/api/org/apiguardian/api/API.html)
- [x] Change is documented in the [User Guide](https://docs.junit.org/snapshot/) and [Release Notes](https://docs.junit.org/snapshot/release-notes.html)

### CODE REVIEW

**Review by mpkorstanje — APPROVED** [review]

File: documentation/modules/ROOT/pages/writing-tests/parallel-execution.adoc [hunk 1]
```diff
@@ -197,9 +213,9 @@ If no configuration strategy is set, JUnit Jupiter uses the `dynamic` configurat
 strategy with a factor of `1`. Consequently, the desired parallelism will be equal to the
 number of available processors/cores.
 
-.Parallelism alone does not imply maximum number of concurrent threads
+.Parallelism alone does not imply the maximum number of concurrent threads
 NOTE: By default, JUnit Jupiter does not guarantee that the number of threads used to
-execute test will not exceed the configured parallelism. For example, when using one
+execute a test will not exceed the configured parallelism. For example, when using one
```
Comment (mpkorstanje) [inline documentation/modules/ROOT/pages/writing-tests/parallel-execution.adoc]:
```suggestion
execute tests will not exceed the configured parallelism. For example, when using one
```

Can't be singular, a test can be execute by at most one thread.
